SKILL.md

```
@@ -0,0 +1,318 @@
+CodeRabbit puts findings in two places — check **both**:
+
+**1. The review summary** (walkthrough + "Actionable comments posted: N" counter):
+
+**GitHub:**
+```bash
+gh api "repos/{owner}/{repo}/pulls/<PR_NUMBER>/reviews" \
+  --jq '[.[] | select(.user.login == "coderabbitai[bot]")] | sort_by(.submitted_at) | last | .body'
+```
+
+The body typically contains a line like:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)